rafthttp/stream.go

```diff
@@ -38,6 +39,16 @@ const (
 	streamBufSize = 4096
 )
 
+var (
+	errUnsupportedStreamType = fmt.Errorf("unsupported stream type")
+
+	streamSupportMap = map[streamType]*semver.Version{
```

Comment:
this is not only ugly but also very inconsistent. 

in 2.0, you might see [app] [2.0-2.0] 
in 2.1, you might see [app] [2.0-2.1]

in my mind, the only way to make it work fine is to make the version the primary index.